kvdb-rocksdb/Cargo.toml

```diff
@@ -13,7 +13,7 @@ harness = false
 
 [dependencies]
 smallvec = "1.0.0"
-fs-swap = "0.2.5"
+fs-swap = "0.2.6"
```

Comment:
this is not counted as a breaking change because it also not publicly exposed?

Reply:
yes, but even if it was, we're bumping the patch version here, so it can't be breaking (assuming `fs-swap` follow semver)